core/packages/test/config/launch-config.toml

```diff
@@ -41,27 +41,92 @@ default_args = [
     name = "rococo-dave"
     validator = true
 
-# Snowbridge
+## Snowbridge
```

Comment:
Rather just delete commented out code. We can always pull it from the version history